packages/alpinejs/src/plugin.js

```diff
@@ -3,5 +3,9 @@ import Alpine from "./alpine";
 export function plugin(callback) {
     let callbacks = Array.isArray(callback) ? callback : [callback]
 
-    callbacks.forEach(i => i(Alpine))
+    callbacks.forEach(i => {
+      if (typeof i === undefined) { throw new Error(`Alpine.plugin(...) expects a callback function but got undefined. Did you misspell your plugin import?`)}
```

Comment:
I don't really think this check is needed at all. What does this really tell someone that the other doesn't?

Reply:
Yeah maybe not, fair enough

Reply:
But you should probably solve that in your tooling. Like mispelling should cause issues the moment you type it. You shouldn't be making it to running in the first place.

You can use strict for JS to yell at you instantly, and you ca n use eslint or typescript to point out you are accessing something that doesn't exist as well.